docs/conf.py

```diff
@@ -21,7 +21,7 @@
 project = 'METplotpy'
 copyright = '2021, NCAR'
 author = 'UCAR/NCAR, NOAA, CSU/CIRA, and CU/CIRES'
-author_list = 'Win-Gildenmeister, M., Burek, T., Fillmore, D., Halley Gotway, J., Adriaansen, D., Fisher, H.'
+author_list = 'Win-Gildenmeister, M., Burek, T., Fillmore, D., Adriaansen, D., Fisher, H.'
```

Comment:
please add Christina Kalb to the list of authors

Reply:
Done